### ISSUE TITLE
DoctrineExtensions\PHPUnit\OrmTestCase class disappeared

### ISSUE THREAD

**Issue by yurevichcv** [beberlei/DoctrineExtensions#142]
Hi, all my tests were based on this class. Why did you remove it from new version of DoctrineExtensions? What should I use instead?

Thanks.

**Comment by stevelacey** [comment]
A lot of stuff was deprecated in `1.0` including the PHPUnit stuff – https://github.com/beberlei/DoctrineExtensions#legacy-versions

I believe there's something comparable in Doctrine core now – @beberlei might know the answer – I did question removing this when I was doing the rest however.

**Comment by jwbrady** [comment]
The main problem is that there is no direct documentation on how to do this, so I initially though it wasn't there. [This blog post](http://resoftsol.com/database-testing-with-phpunit/) gives some hints on how to do it. What I figured out is:

1. Create your own PDO and give it to PHPUnit (using getconnection) and doctrine ([using the `pdo` config argument](http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html))
2. Create your DB Schema in [setUpBeforeClass](https://phpunit.de/manual/current/en/fixtures.html#fixtures.more-setup-than-teardown)

This mostly works. One issue I found is that without something like the TestConnection class (and probably others) to wrap the Doctrine DBAL connection, DBUnit will use straight SQL and PDO to insert/query data sets.  So any thing that depends on Doctrine won't work (such as quoted column names).
